Makefile

```diff
@@ -81,6 +81,15 @@ clusterawsadm: dep-ensure ## Build clusterawsadm binary.
 cluster-api-dev-helper: dep-ensure ## Build cluster-api-dev-helper binary
 	bazel build //hack/cluster-api-dev-helper $(BAZEL_ARGS)
 
+.PHONY: release-binaries
+release-binaries: ## Build release binaries
+	bazel build --platforms=@io_bazel_rules_go//go/toolchain:linux_amd64 //cmd/clusterctl //cmd/clusterawsadm
+	bazel build --platforms=@io_bazel_rules_go//go/toolchain:darwin_amd64 //cmd/clusterctl //cmd/clusterawsadm
+	install bazel-bin/cmd/clusterawsadm/darwin_amd64_pure_stripped/clusterawsadm clusterawsadm-darwin-amd64
+	install bazel-bin/cmd/clusterawsadm/linux_amd64_pure_stripped/clusterawsadm clusterawsadm-linux-amd64
+	install bazel-bin/cmd/clusterctl/darwin_amd64_pure_stripped/clusterctl clusterctl-darwin-amd64
+	install bazel-bin/cmd/clusterctl/linux_amd64_pure_stripped/clusterctl clusterctl-linux-amd64
```

Comment:
the root seems like a weird place to put these binaries

Reply:
fixed, thanks!